### PR TITLE
WebHeaderUI Wallet address UI 

### PR DESCRIPTION
**Pull request by CompositeFellow**
Added all of the UI elements for the wallet address addition. https://app.asana.com/0/1201944923757693/1202029268475923/f
image
image
image
image

### CODE REVIEW

File: packages/react-app/src/components/global/Header/UserMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { toggleDiscordSignIn } from '@app/services/discord.service';
+import React from 'react';
+
+const UserMenu = () => {
+	const { data: session } = useSession({ required: false });
+	const { isOpen, onOpen, onClose } = useDisclosure();
+	return (
+		<Menu>
+			<MenuButton as={'button'}>
+				<Avatar src={session?.user?.image ?? ''} mr={'.25em'} size={'md'} />
```
Comment (jordaniza) [inline packages/react-app/src/components/global/Header/UserMenu.tsx:26]: If the avatar is not found, would it be better to render a placeholder, or not render the avatar at all

File: packages/react-app/src/components/global/Header/UserMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			</MenuList>
+			{/* Add Wallet Address Modal */}
+			<Modal isOpen={isOpen} onClose={() => onClose()} isCentered>
+				<ModalOverlay />
+				<ModalContent>
+					<ModalHeader textAlign={'center'}>Enter Wallet Address</ModalHeader>
+					<ModalBody>
+						<Input />
+						<Button my={'1em'} w={'100%'}>
+              Save Address
```
Comment (jordaniza) [inline packages/react-app/src/components/global/Header/UserMenu.tsx:40]: I suppose this is WIP

File: packages/react-app/src/components/global/Header/UserMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+						</Button>
+					</ModalBody>
+				</ModalContent>
+			</Modal>
+		</Menu>
+	);
+};
+
+export default UserMenu;
```
Comment (jordaniza) [inline packages/react-app/src/components/global/Header/UserMenu.tsx:49]: Would recommend some basic tests to check rendering in different session states

File: packages/react-app/src/components/global/Header/UserMenu.tsx (hunk 2 above)
Comment (jordaniza) [reply packages/react-app/src/components/global/Header/UserMenu.tsx:40]: Saw your discord comment

File: packages/react-app/src/components/global/Header/UserMenu.tsx (hunk 1 above)
Comment (CompositeFellow) [reply packages/react-app/src/components/global/Header/UserMenu.tsx:26]: I'll add a comment here but the Avatar loads a default blank user pic when the discord user image doesn't exist or fails to load.

**Pull request closed without merging**